add email verification details to scim azure (#20577)
DataDog/documentation: content/en/account_management/scim/_index.md

To avoid losing access to your data, Datadog strongly recommends that you create a [service account][3] dedicated to SCIM. Within that service account, create an application key to use in the SCIM integration.

## Email verification

> Creating a new user with SCIM will trigger an email to the user. For first time accessm it is a required that the user logs in using the invite link shared by email. The link stays active for 30 days, if it expires, you can go to the [user settings page][6] and select a user to resend an invite link.

Reviewer: ```suggestion
Creating a new user with SCIM triggers an email to the user. For first time access, you are required to login through the the invite link shared by email. The link is active for 30 days. If it expires, go to the [user settings page][6] and select a user to resend an invite link.
```

[1]: /account_management/scim/#using-a-service-account-with-scim
[2]: https://app.datadoghq.com/organization-settings/application-keys
[3]: /account_management/org_settings/service_accounts
[4]: /account_management/api-app-keys
[5]: /account_management/scim/#email-verification

> [6]: /organization-settings/users

Reviewer: ```suggestion
[6]: https://app.datadoghq.com/organization-settings/users
```
